src/pcp/gadgets.rs

```diff
@@ -274,11 +266,10 @@ impl<F: FieldElement> MeanVarUnsigned<F> {
         Ok(())
     }
 
-    pub(crate) fn call_poly_fft(
-        &mut self,
-        outp: &mut [F],
-        inp: &Vec<Vec<F>>,
-    ) -> Result<(), PcpError> {
+    // The single character variable names used in this function are consistent
```

Comment:
I don't think this comment is necessary, but I don't object to it.

Reply:
My thought is that any time we opt out of any kind of static analysis we should explain why, but I haven't done that consistently, and at some point it's more noise than it's worth, so I deleted the comment.